### PR TITLE
[quantization] Make Attention Mask Fill Value Configurable

### PR DESCRIPTION
**Pull request by dvsav**
# What

This change introduces a global configuration parameter attention_mask_fill_value to PTQConfig in order to make the attention mask fill value configurable across the TICO codebase.

# Why

The detailed motivation behind this change is described in issue [\[quantization\] Make Attention Mask Fill Value Configurable & Run Ablation Study](https://github.com/Samsung/TICO/issues/643).

# Implementation Details

### 1. Added `attention_mask_fill_value` to `PTQConfig` (`tico/quantization/config/ptq.py`)
- Added new field `attention_mask_fill_value: float = -120.0` with documentation
- Updated `child()` method to propagate this value to child configurations

### 2. Updated Wrapper Classes to Use Configurable Value

**Llama Wrappers:**
- `QuantLlamaAttention` (`tico/quantization/wrapq/wrappers/llama/quant_attention.py`):
  - Causal mask template creation
  - Boolean/int mask to additive mask conversion
  - Clamp operations

- `QuantLlamaDecoderLayer` (`tico/quantization/wrapq/wrappers/llama/quant_decoder_layer.py`):
  - Causal mask template creation
  - Boolean mask to additive mask conversion
  - Clamp operations

- `QuantLlamaModel` (`tico/quantization/wrapq/wrappers/llama/quant_model.py`):
  - Causal mask template creation
  - Boolean/int mask to additive mask conversion
  - Clamp operations

**Qwen VL Wrappers:**
- `QuantQwen3VLTextAttention` (`tico/quantization/wrapq/wrappers/qwen_vl/quant_text_attn.py`):
  - Causal mask template creation

- `QuantQwen3VLTextModel` (`tico/quantization/wrapq/wrappers/qwen_vl/quant_text_model.py`):
  - Causal mask template creation
  - 2D and 4D mask handling with clamp operations

**Fairseq Wrappers:**
- `QuantFairseqDecoder` (`tico/quantization/wrapq/wrappers/fairseq/quant_decoder.py`):
  - Now uses `self.qcfg.attention_mask_fill_value` instead of hardcoded value

- `QuantFairseqMultiheadAttention` (`tico/quantization/wrapq/wrappers/fairseq/quant_mha.py`):
  - Updated to use PTQConfig's value as default, with option to override via constructor parameter

# Usage Example

```python
from tico.quantization.config.ptq import PTQConfig
from tico.quantization.wrapq.dtypes import DType

# Use default value (-120.0)
cfg = PTQConfig(default_dtype=DType.uint(8))

# Or specify a custom value
cfg = PTQConfig(
    default_dtype=DType.uint(8),
    attention_mask_fill_value=-100.0  # Custom value
)
```

The value is automatically propagated to all child configurations via the `child()` method, ensuring consistent behavior across the entire model hierarchy.

### CODE REVIEW

**Review by mhs4670go — APPROVED** [review]
LGTM
